tests/acceptance-bdd/test_literature_suggestion_form.py

```diff
@@ -0,0 +1,287 @@
+def test_wrong_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing the correct DOI')
+def test_correct_doi():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a wrong url')
+def test_wrong_url():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a wrong thesis date')
+def test_wrong_thesis_date():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a correct thesis date')
+def test_correct_thesis_date():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Writing a correct url')
+def test_correct_url():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Submit thesis and verify record detail in the list page')
+def test_verify_thesis_record_in_the_list_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Submit thesis and verify record detail in the detail page')
+def test_verify_thesis_record_in_the_detail_page():
+    pass
+
+
+@scenario('literature_suggestion.feature', 'Accept thesis and verify the confirmation message')
+def test_verify_accepted_thesis():
+    pass
+
+
+# when statements for create literature page
+@when('I go to the literature suggestion page')
+def go_to_create_literature():
+    Arsenic().get(os.environ['SERVER_NAME'] + '/literature/new')
+
+
+@when('I click on the submit button')
+def click_submit_button():
+    Arsenic().get_element_with_locator(CL.SUBMIT_BUTTON).click()
+
+
+@when('I click on the accept button')
+def click_accept_button():
+    Arsenic().get_element_with_locator(CL.ACCEPT_BUTTON).click()
+
+
+@when('I click on the button Skip, and uncollapse all panels')
+def click_skip_and_fill_manually():
+    Arsenic().hide_title_bar()
+    Arsenic().get_element_with_locator(CL.SKIP_IMPORT_DATA, waiting_time=5).click()
+
+    WebDriverWait(Arsenic(), 10).until(
+        EC.text_to_be_present_in_element(
+            CL.LITERATURE_FORM,
+            'Type of Document',
+        )
+    )
+
+    Arsenic().get_element_with_locator(CL.CONFERENCE_TAB).click()
+    Arsenic().get_element_with_locator(CL.PROCEEDINGS_TAB).click()
+    Arsenic().get_element_with_locator(CL.REFERENCES_TAB).click()
+    Arsenic().get_element_with_locator(CL.COMMENTS_TAB).click()
+
+    Arsenic().show_title_bar()
+
+
+@when(parsers.cfparse('I select the value {input_select} in the subject dropdown list'))
+def select_value_in_subject_multiselect_widget(input_select):
+    Arsenic().get_element_with_locator(CL.DROPDOWN_BUTTON).click()
+    Arsenic().find_element_by_xpath(
+        '//input[@value="' + input_select + '"]'
+    ).click()
+    Arsenic().get_element_with_locator(CL.DROPDOWN_BUTTON).click()
+
+
+# when statements for holdingpen pages
+@when('I go to the holding panel list page')
+def go_to_holding_pen_literature_list():
+    Arsenic().get(os.environ['SERVER_NAME'] + '/holdingpen/list/?page=1&size=10&status=HALTED&workflow_name=HEP')
+
+
+@when('I should see the record in the list page')
```

Comment:
By default pytest-bdd does not allow to mix the syntax of the steps, that should remain always the following:

- Given
- Then
- When

buuuuuut actually I discovered this PR: https://github.com/pytest-dev/pytest-bdd/pull/130 where they have added a way to relax the strict Gherkin syntax. This allows to mix different steps even thought they do not follow the classic order `Given when then`. 
 
What do you think? Should we allow to use a mixed syntax? 

IMHO YES! Morover in this issue https://github.com/pytest-dev/pytest-bdd/issues/118 a guy that was writing selenium test sad that was necessary for this kind of tests use a mixed syntax.